metricflow/engine/metricflow_engine.py

```diff
@@ -91,38 +96,39 @@ class MetricFlowQueryRequest:
     time_constraint_end: Get data for the end of this time range.
     where_constraint: A SQL string using group by names that can be used like a where clause on the output data.
     order_by_names: metric and group by names to order by. A "-" can be used to specify reverse order e.g. "-ds"
+    order_by: metric and group by objects to order by
```

Comment:
Kind of unrelated, but it seems weird to pass in parameters that are called `group by` into an `order_by`. I noticed that this is the case in GQL, not just this comment.

Also, it looks like I forgot to add a comment here for `group_by` & `metrics` too. Do you mind adding it?